src/core/EdFi.DataManagementService.Core/Handler/UpdateByIdHandler.cs

```diff
@@ -45,14 +45,14 @@ public async Task Execute(PipelineContext context, Func<Task> next)
 
         context.FrontendResponse = result switch
         {
-            UpdateSuccess
+            UpdateSuccess updateSuccess
```

Comment:
SonarLint suddenly started yelling at me that we have a redundant cast here. That's true, but I don't know why it didn't throw a warning/error about this before.

Reply:
Never saw the warning myself either but good change.